src/process_collector.rs

```diff
@@ -144,6 +156,11 @@ impl Collector for ProcessCollector {
         // memory
         self.vsize.set(p.stat.vsize as f64);
         self.rss.set(p.stat.rss as f64 * *PAGESIZE);
+        if let Ok(status) = p.status() {
+            if let Some(size) = status.vmsize {
```

Comment:
Shouldn’t this be vmswap instead of vmsize ?

Reply:
🤦‍♂️ yes. Fixed.